### ISSUE TITLE
/typings/index.d.ts(293,1): error TS1046: A 'declare' modifier is required for a top level declaration in a .d.ts file.

### ISSUE THREAD

**Issue by TiFu** [cnguy/kayn#61]
Hey,

first: thanks for creating this great API!

I am currently trying to get kayn to work with TypeScript. Unfortunately, it looks like the typings are incorrect for TypeScript v3.2.2 and are missing multiple `declare` modifiers in front of top level classes/enums.

Repo for reproduction: https://github.com/TiFu/kayn-bug
Output of `npm run build`:
```
$ npm run build

> test@1.0.0 build ~\test
> tsc

node_modules/kayn/typings/index.d.ts(293,1): error TS1046: A 'declare' modifier is required for a top level declaration in a .d.ts file.
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! test@1.0.0 build: `tsc`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the test@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     ~\AppData\Roaming\npm-cache\_logs\2018-12-31T02_14_58_111Z-debug.log

```

A trivial fix would be prepending `declare` in front of all top level classes/enums in `typings/index.d.ts` and upgrading the TypeScript version in `typings/package.json`.

**Comment by cnguy** [comment]
Thank you! Forgot about TS 3.. releasing the fix after I double-check

**Comment by cnguy** [comment]
Fix is in v0.9.3, try it out

**Comment by TiFu** [comment]
Works like a charm. Thanks!

Happy new year!
